src/DomCrawler/Link.php

```diff
@@ -54,6 +55,6 @@ protected function setNode(\DOMElement $node)
 
     protected function getRawUri()
     {
-        return $this->element->getAttribute('href');
+        return $this->element->getAttribute('href') ?? '';
```

Comment:
Shouldn't it return the current URI?
```suggestion
        return $this->element->getAttribute('href') ?: $this->currentUri;
```

Reply:
Honestly, I don't know.